ProjectEureka/project-eureka-flutter
# PE-54 | Implmented universal styling theme

**What I did:**

- Created a new dart file called styles.dart.
- The file contains objects of ThemeData, where you can change parameters such as scaffoldbackgroundColor and primaryColor. 

**How to test:**

1.  When you first open the app, the default colors for our app will be the ones we have been working with.
2. In order to switch to dark mode open settings on your phone and under display switch to dark theme. ![image](https://user-images.githubusercontent.com/38572314/111588731-2cf95200-8781-11eb-9736-4e773aabed23.png)
3. Make sure the colors on the **EurekaAppbar()** and background change across all pages.
4. Feel free to mess around with the given parameters by entering the hex color of your choice and make sure it changes.
